src/components/document/DocumentTabDetails.vue

```diff
@@ -175,10 +175,16 @@ export default {
           }
         },
         {
-          name: 'metadata.tika_metadata_resourcename',
-          label: this.$t('document.name'),
-          trClass: 'document__content__basename',
-          value: this.document.meta('resourcename')
+          name: 'title',
+          label: this.$t('document.title'),
+          trClass: 'document__content__title',
+          value: this.document.title
+        },
+        {
```

Comment:
I don't think we need to display the `titleNorm`, it is only for sorting.

Reply:
Indeed !